omero/sysadmins/version-requirements.rst

```diff
@@ -561,7 +574,7 @@ OMERO support policies
       - to Jan 2020
       - |Recommended|
       - |Recommended|
-      - |Recommended|
+      - |Deprecated|
```

Comment:
Potentially even broken, but I'm happy to take responsibility for elevating that when the time comes.